doc/jsk_pcl_ros/nodes/euclidean_clustering.md

```diff
@@ -48,6 +53,29 @@ sensor_msgs/PointCloud2[] output
 
    The minimum number of the points of one cluster.
 
+* `~multi` (Boolean, default: `false`)
+
+   Flag of multi euclidean clustering. If `~multi` is `true`, synchronized `~input` and `~input/indices` are used.
+
+* `~approximate_sync` (Boolean, default: `False`):
+
+   Policy of synchronization, if `false` it synchornizes exactly, else approximately.
+   This value is only valid in case of `~multi` is `true`.
+
+* `~queue_size` (Int, default: `1`):
```

Comment:
Default: `1` for synchronizer is not kindful ...
How about 10 or 20?

Reply:
I changed default `queue_size` to `20` and modified docs too.